src/app/series/directives/series-templates.component.ts

```diff
@@ -1,11 +1,56 @@
 import { Component, OnDestroy } from '@angular/core';
 import { Subscription } from 'rxjs';
-import { SeriesModel, TabService } from '../../shared';
+import {
+  SeriesModel,
+  TabService,
+  AudioVersionTemplateModel,
+  AudioFileTemplateModel
+} from '../../shared';
 
 @Component({
-  styleUrls: [],
+  styleUrls: ['series-templates.component.css'],
   template: `
-    <h1>These are some templates</h1>
+    <form *ngIf="series">
+
+      <template ngFor let-v [ngForOf]="series.versionTemplates">
+        <div *ngIf="!v.isDestroy" class="version">
+          <publish-fancy-field textinput required [model]="v" name="label" label="Version Label">
+            <div class="actions">
+              <a (click)="removeVersion(v)"><i class="icon-cancel"></i>Remove Template</a>
+            </div>
+          </publish-fancy-field>
+
+          <publish-fancy-field class="length" [model]="v" label="Total length in seconds" invalid="lengthAny">
+            <publish-fancy-field number small inline hideinvalid [model]="v" name="lengthMinimum" label="Minimum">
+            </publish-fancy-field>
+            <publish-fancy-field number small inline hideinvalid [model]="v" name="lengthMaximum" label="Maximum">
+            </publish-fancy-field>
+          </publish-fancy-field>
+
+          <publish-fancy-field label="Audio Segments">
+            <div class="actions">
+              <a *ngIf="canAddFile(v)" (click)="addFile(v)"><i class="icon-plus"></i>Add Segment</a>
+              <a *ngIf="canRemoveFile(v)" (click)="removeFile(v)"><i class="icon-cancel"></i>Remove Segment</a>
```

Comment:
I'd like to revisit after I do this audio-uploader-revamp PR.  It's sort of problematic to go changing the number of segments after you've started creating stories against that template.  So this template-editor may end up being more of a "template-creator-only".  Or maybe you can change the durations/labels after creation, but not the number of segments.